polaris/polaris/tests/processes/test_create_stellar_deposit.py

```diff
@@ -173,8 +173,5 @@ def test_deposit_stellar_no_trustline(acc1_usd_deposit_transaction_factory):
     deposit = acc1_usd_deposit_transaction_factory()
     deposit.status = Transaction.STATUS.pending_anchor
     deposit.save()
-    assert not create_stellar_deposit(deposit)
-    assert (
-        Transaction.objects.get(id=deposit.id).status
-        == Transaction.STATUS.pending_trust
-    )
+    assert create_stellar_deposit(deposit)
```

Comment:
Actually, I don't see a function that extracts the ID from result XDR, did you want to wait off on implementing it?

Reply:
Oh maybe I misunderstood from our sync, I reverted the changes that created the claimable_balance_id column.
I was under the assumption that we'll push that responsibility to the wallet since they can decode and parse the results xdr themselves.

Reply:
I can make an assert here to decode the XDR and check if there was a claimable balance operation made.

Reply:
Yea I don't think we should have it returned in the `/transactions` JSON but I think it should still be in the data model.

Reply:
Got it, ok I'll make these changes then.